test/unit/connection.js

```diff
@@ -44,6 +44,18 @@ describe('connection', function () {
 
 	after(() => wss.close());
 
+	describe('Connection tokenProvider', () => {
+		it('should handle token provider rejection ', async () => {
+			const client = new StreamChat('apiKey', {
+				allowServerSideConnect: true,
+				baseURL: 'http://localhost:9999', // invalid base url
```

Comment:
```suggestion
```
Not sure why this is needed, could you elaborate?

Reply:
I was not sure what the required parameters were for a proper `StreamChat` instantiation. If it works without it, we can remove it; iirc I just copy/pasted this from another test in this file :sweat_smile: